starfish/core/util/test/test_dtype.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import pytest
+import xarray as xr
+
+from ..dtype import preserve_float_range
+
+
+def combos(make_xarrays: bool):
```

Comment:
What if someone passed an integer dtype? or float64 that's also below zero? Might be worth a few more test combinations.

Reply:
sure, i can add some examples of that.